### ISSUE TITLE
Should not lint on box-imported R6 object instantiation

### ISSUE THREAD

**Issue by radbasa** [Appsilon/box.linters#28]
## box.linters version

0.0.0.9006

## Source code to lint

```R
box::use(
  path/to/module_r6[SomeClass],
)

s <- SomeClass$new()
```

## Lint command

```R
lintr::lint(code, linters = lintr::linters_with_defaults(defaults = box.linters::rhino_default_linters))
```

## Lint result
```
<text>:6:6: warning: [box_usage_linter] <package/module>$function does not exist.
s <- SomeClass$new()
     ^~~~~~~~~~~~~
```

## Related issues
- #22 

## Expected result

Should not lint on `R6$new()`.
